### DISCUSSION TITLE
Links are blue. You can click on links. Don't break this common functionality :)

### DISCUSSION THREAD

**Discussion by Poopooracoocoo** [community/community#6147] [Other Feature Feedback, Questions, & Ideas] [answered]
Links are blue.
That's how you know you can click on them. 

The old layout was also somehow just nicer to read. I haven't taken the time to pinpoint why however.

**Comment by MylesBorins** [comment, marked as answer]
Which links are you referring to?

The title of the release is now black but blue on hover, which is a pattern used across the site such as the links for the titles of issues in the issue title list

https://github.com/jstime/jstime/issues
